src/app/containers/README.md

```diff
@@ -0,0 +1,7 @@
+This repo follows the container component pattern.
```

Comment:
It would improve readability to have a heading at the top of each readme file.

Reply:
The heading just being the name of the directory?

Reply:
Yes.

Reply:
Done